examples/generative/ddpm.py

```diff
@@ -0,0 +1,682 @@
+"""
+Title: Denoising Diffusion Probabilistic Model
+Author: [A_K_Nain](https://twitter.com/A_K_Nain)
+Date created: 2022/11/30
+Last modified: 2022/11/30
+Description: Generating images of flowers with denoising diffusion probabilistic models.
+"""
+
+"""
+## Setup
```

Comment:
Please add an introduction section explaining what this example is about, what the reader will learn, and why this is valuable.